ttg/ttg/parsec/ttg.h

```diff
@@ -2206,6 +2240,10 @@ namespace ttg_parsec {
       }
     }
 
+    static parsec_key_t make_key(const parsec_taskpool_t *tp, const parsec_assignment_t *as) {
```

Comment:
I'm trying to understand what `make_key` does... `parsec_assignment_t` is a struct with one integer member. How is it safe to cast it to a `uintptr_t` below?

Reply:
`self.locals` is an array of `assignment_t` with more than 4 of them, so there is room in `self.locals` to store the pointer-size int. Not clean, but I'm abusing the only space I have to store stack information.